src/commands/schema_diff.ts

```diff
@@ -0,0 +1,85 @@
+import { BranchScopeProps } from '../types';
+import { createPatch } from 'diff';
+import { Database } from '@neondatabase/api-client';
+import chalk from 'chalk';
+import { writer } from '../writer.js';
+import { branchIdFromProps, branchIdResolve } from '../utils/enrichers.js';
+
+type SchemaDiffProps = BranchScopeProps & {
+  branch?: string;
+  baseBranch?: string;
```

Comment:
you can name the parameter to "branch" to avoid confusions

Reply:
hmm, i see that is is named "branch", what is "baseBranch" then

Reply:
Why not call it "base-branch" in options?

Reply:
Corrected it in command.